site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+        <div class="container topnav">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <img src="../img/eclipse-collections-logo.png" style="width:175px;height:50px;" align="left" alt="Eclipse Collections">
+            </div>
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <a href="#start">Kom i gang</a>
+                    </li>
+                    <li>
+                        <a href="#concept">Konsepter</a>
+                    </li>
+                    <li>
+                        <a href="#learn">Lær</a>
+                    </li>
+                    <li>
+                        <a href="#history">Historikk</a>
+                    </li>
+                    <li>
+                        <a href="#refGuide">Veiledning</a>
+                    </li>
+                    <li>
+                        <a href="#sourceCode">Kildekode</a>
+                    </li>
+                    <li>
+                        <a href="#contribute">Bidra</a>
+                    </li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                           aria-expanded="false">Javadoc<span class="caret"></span></a>
+                        <ul class="dropdown-menu">
+                            <li><a href="../javadoc/10.0.0">10.0.0</a></li>
+                            <li><a href="../javadoc/9.2.0">9.2.0</a></li>
+                            <li><a href="../javadoc/9.1.0">9.1.0</a></li>
+                            <li><a href="../javadoc/9.0.0">9.0.0</a></li>
+                            <li><a href="../javadoc/8.2.0">8.2.0</a></li>
+                            <li><a href="../javadoc/8.1.0">8.1.0</a></li>
+                            <li><a href="../javadoc/8.0.0">8.0.0</a></li>
+                            <li><a href="../javadoc/7.1.0">7.1.0</a></li>
+                        </ul>
+                    </li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                           aria-expanded="false">Språk<span class="caret"></span></a>
+                        <ul class="dropdown-menu">
```

Comment:
The paths need to be relative.
```
<ul class="dropdown-menu">
                        <li><a href="../index.html">English</a></li>
                        <li><a href="../de/index.html">Deutsch</a></li>
                        <li><a href="../el/index.html">Ελληνικά</a></li>
                        <li><a href="../es/index.html">Español</a></li>
                        <li><a href="../cn/index.html">中文</a></li>
                        <li><a href="">Français</a></li>
                        <li><a href="../ja/index.html">日本語</a></li>
                        <li><a href="../pt-br/index.html">Português-Brasil</a></li>
                        <li><a href="../ru/index.html">Русский</a></li>
                        <li><a href="../hi/index.html">हिंदी</a></li>
                    </ul>
```

Reply:
Good catch! Thanks! Using the English version as a starting point was not so good idea. Had to fix paths several other places too. 😃